frontend/src/bundles/studio/components/video-menu/components/menu-content/script-content/components/script.tsx

```diff
@@ -0,0 +1,71 @@
+import { Editable, EditablePreview, EditableTextarea } from '@chakra-ui/react';
```

Comment:
import, please, from components instead of directly from the chakra

Reply:
Done